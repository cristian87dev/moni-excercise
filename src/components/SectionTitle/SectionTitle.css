.section-title {
  margin: 2rem 0;
  color: var(--font-color);
}
