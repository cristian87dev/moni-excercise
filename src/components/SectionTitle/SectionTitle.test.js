import React from 'react';
import renderer from 'react-test-renderer';
import SectionTitle from './index';

it('SectionTitle renders without crashing', () => {
  const component = renderer.create(
    <SectionTitle>Test Content</SectionTitle>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
