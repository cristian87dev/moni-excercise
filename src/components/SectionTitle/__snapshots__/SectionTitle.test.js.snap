// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle renders without crashing 1`] = `
<h2
  className="section-title"
>
  Test Content
</h2>
`;
